comms/src/peer_manager/identity_signature.rs

```diff
@@ -38,9 +39,11 @@ use crate::{
 };
 
 /// Signature that secures the peer identity
-#[derive(Debug, Clone, Serialize, Deserialize, PartialEq, Eq)]
+#[derive(Derivative, Clone, Serialize, Deserialize, PartialEq, Eq)]
+#[derivative(Debug)]
 pub struct IdentitySignature {
     version: u8,
+    #[derivative(Debug = "ignore")]
```

Comment:
Signature contains public values (SecretKey is a public scalar `s` in this case) so can remove this ignore

Reply:
Done.